_posts/2022-10-10-1.16-announcement.md

```diff
@@ -0,0 +1,401 @@
+---
+layout: post
+title:  "Announcing the Release of Apache Flink 1.16"
+subtitle: ""
+date: 2022-10-10T08:00:00.000Z
+categories: news
+authors:
+- godfreyhe:
+  name: "Godfrey He"
+  twitter: "godfreyhe"
+
+---
+
+Apache Flink continues to grow at a rapid pace and is one of the most active 
+communities in Apache. Flink 1.16 had over 230 contributors enthusiastically participating, 
+with 19 FLIPs and 900+ issues completed, bringing a lot of exciting features to the community.
+
+Flink has become the leading role and factual standard of stream processing, 
+and the concept of the unification of stream (aka unbounded) and batch (aka bounded) data 
```

Comment:
```suggestion
Flink has become the leading role and factual standard of stream processing. The concept of the unification of stream and batch data 
```
I think a @knaufk had a good point here about not mixing up the concepts of bound/unbound data vs processing types, i.e. stream and batch processing, in [his comment in the GoogleDocs draft](https://docs.google.com/document/d/1rIBNpzJulqEKJCuYtWtG-vDmSsGpN9sip_ewpzMequ0/edit?disco=AAAAf9slSt0) WDYT?

Reply:
make sense